setup.py

```diff
@@ -59,10 +59,6 @@ def read_file(filename):
       author_email='alexis@notmyidea.org',
       url='https://github.com/spiral-project/ihatemoney',
       packages=find_packages(),
-      package_data={'': ['*.rst', '*.py', '*.yaml', '*.po', '*.mo', '*.html',
```

Comment:
And I finally didn't remove the ``include_package_data`` entry as it is not clear if it used or not ...even though its works without